src/Controllers/Version2/class-wc-rest-system-status-tools-v2-controller.php

```diff
@@ -538,6 +551,11 @@ public function execute_tool( $tool ) {
 				$message = __( 'Database upgrade routine has been scheduled to run in the background.', 'woocommerce-rest-api' );
 				break;
 
+			case 'clear_template_cache':
+				wc_clear_template_cache();
```

Comment:
we should add a function check for `wc_clear_template_cache` here and return an error if it does not exists.